modules/migrations/github.go

```diff
@@ -301,10 +301,15 @@ func (g *GithubDownloaderV3) convertGithubRelease(rel *github.RepositoryRelease)
 			Created:       asset.CreatedAt.Time,
 			Updated:       asset.UpdatedAt.Time,
 			DownloadFunc: func() (io.ReadCloser, error) {
+				g.sleep()
 				asset, redir, err := g.client.Repositories.DownloadReleaseAsset(g.ctx, g.repoOwner, g.repoName, *asset.ID, http.DefaultClient)
 				if err != nil {
 					return nil, err
 				}
+				err = g.RefreshRate()
+				if err != nil {
+					log.Error("g.client.RateLimits: %s", err)
+				}
```

Comment:
doesn't g.sleep() do this?

Reply:
No, it will not. `g.sleep()` is not enough. we have to refresh the rate again after we invoked the github API.